samples/bluetooth/radio_coex_1wire/src/main.c

```diff
@@ -78,11 +87,15 @@ static void check_input(void)
 
 	if ((input_char == 'g') ^ APP_GRANT_ACTIVE_LOW) {
 		nrf_gpio_pin_set(APP_GRANT_GPIO_PIN);
+		state_granted = true;
```

Comment:
I see you're using this only for printing the current state, is there any reason to not just print the status here in this `if/else if` ?